### ISSUE TITLE
3.0.3 devns error adding a page

### ISSUE THREAD

**Issue by CanRau** [ryancramerdesign/ProcessWire#1594]
Just got this error message on a white screen while trying to add a new page under Home.

Figured the reason and it's maybe not the most usual case but think it should be noted.

I created a new template and mixed things a little so while I creating the page I let PW copy the fields of the "user" template

Then I hit new page in the page tree from the homepages action menu.
Entered Title and selected the newly created template, after hitting save I got this error

```
Notice: Trying to get property of non-object in /Users/admin/Can/htdocs/processwire/happyfamily/wire/modules/Process/ProcessPageAdd/ProcessPageAdd.module on line 626

Fatal error: Call to a member function getInputfield() on null in /Users/admin/Can/htdocs/processwire/happyfamily/wire/modules/Process/ProcessPageAdd/ProcessPageAdd.module on line 627
```

After restoring from backup and creating the template with just a title field everything worked as expected

__edit__
error came up again. I set up second template and parent for users, until now I just changed the template and parent of the superuser, but now wanted to create a new user with custom template under custom parent and the same error showed up..what am I missing?

**Comment by ryancramerdesign** [comment]
Thanks @CanRau I'm not exactly sure I know how to reproduce this one, so I've gone ahead and taken a swat at the error messages you mentioned… adding an additional check to make sure $field was available before trying to access it. I think this may only be duct tape, but curious to know what changes you see from it?

**Comment by CanRau** [comment]
How fast! :D

Okay, the error is gone, but I got a dialog for a normal page, even though it says "member" template in the top (I only allowed this single template for the parent). By normal I mean I get a translatable title and name field
title field isn't present on this template and "Desregard global fields" is checked, and the name shouldn't be translatable either..
Just realized the same behavior when trying to add a "normal" user from page tree under access>users

Using /access/users/add it works now :)

Don't know why but first I wasn't able to name this new user "jennifer"
he told me there is already a page named jennifer which isn't true, "jen" worked and now I could change to jennifer..but it's fine for now..

Thank you! :D fastest support ever! I'm really excited to work with PW3 for the first time now..Of course I don't know exactly how long development will take but as soon as possible it goes online. It's a private community though, so I would give feedback and maybe talk about it.. ;-)

Cheers
